src/aequitas/bias.py

```diff
@@ -184,16 +324,335 @@ def get_disparity_predefined_groups(self, df, ref_groups_dict, key_columns=None,
+
+        :return: Dictionary indicating whether each group has equal variance
+        (in comparison with reference group)
+        '''
+
+        # Immediately set ref_group status for equal variance (with itself): True
+
+        eq_variance = {ref_group: True}
+
+        for attr_value, sample in sample_dict.items():
+            # make default normality_p value (only used when len(sample) < 8)
+            # large enough that it is always greater than alpha
+            normality_p = pd.np.inf
+
+            # skew test requires at least 8 samples
+            if len(sample) >= 8:
+                _, normality_p = stats.normaltest(sample, axis=None, nan_policy='omit')
+
+            # if tested normality is False or less than 8 samples, use levene
+            # test to check equal variance between groups
+            if normality_p < alpha or len(sample) < 8:
+                # if ref_group is not normal, can't use f-test or bartlett test
+                # for any samples, so check for equal varience against ref_group
+                # using levene test for all groups and return dict
+                # (since includes all groups)
+                if attr_value == ref_group:
+                    for group, sample_list in sample_dict.items():
+                        _, equal_variance_p = stats.levene(sample_dict[ref_group],
+                                                           sample_list,
+                                                           center='median')
+                        if equal_variance_p < alpha:
+                            eq_variance[group] = False
+                        else:
+                            eq_variance[group] = True
+
+                    return eq_variance
+
+                # if a non-ref group is not normal, can't use f-test or bartlett
+                # for that group, check for equal variance (against ref_group)
+                # using levene test and add result to dictionary
+                _, equal_variance_p = stats.levene(
+                    sample_dict[ref_group], sample, center='median')
+
+                if equal_variance_p < alpha:
+                    eq_variance[attr_value] = False
+                else:
+                    eq_variance[attr_value] = True
+
+        # for all normally distributed non-ref groups, use bartlett test to
+        # check for equal variance (against ref_group). Add results to dict
+        untested_groups = sample_dict.keys() - eq_variance.keys() - set(ref_group)
+        untested = {key: val for (key, val) in sample_dict.items() if key in untested_groups}
+        for sample, sample_list in untested.items():
+            _, equal_variance_p = stats.bartlett(sample_dict[ref_group], sample_list)
+            if equal_variance_p < alpha:
+                eq_variance[attr_value] = False
+            else:
+                eq_variance[attr_value] = True
+
+        return eq_variance
+
+    @classmethod
+    def check_alpha(cls, x, alpha=5e-2):
```

Comment:
this appears to be a staticmethod or a module-level helper function, not a classmethod.

also, though, it's *really* simple. I 100% believe that it might be helpful ... but, can someone not say:

```
x < alpha
```

or:

```
lambda x: x < ...
```

Reply:
agreed re: calling lambda. actually made the mask w/o this, so think it's okay to go ahead and remove fcn completely